src/main/java/dan200/computercraft/shared/peripheral/monitor/TileMonitor.java

```diff
@@ -526,6 +532,16 @@ private boolean mergeDown()
         return true;
     }
 
+    void updateNeighborsNextTick() {
+        needsUpdate = true;
```

Comment:
Do we also need a call to `TickScheduler.schedule( this );`, or does that kinda magically happen due to `onLoad` or similar?

Reply:
I don’t think it’s necessary, seems to work without it.